src/main.ts

```diff
@@ -1,21 +1,19 @@
 import * as core from '@actions/core';
 import * as installer from './installer';
+import fs from 'fs';
 import * as auth from './authutil';
 import * as path from 'path';
 import {restoreCache} from './cache-restore';
 import {URL} from 'url';
-import os = require('os');
```

Comment:
there was an explicit reason why we did this.  It was because one of the dependencies in toolkit did this and some of the testing mock injection wasn't working.  if that's not the case then ok.

Reply:
👍🏾 Good to know.  As far as I know, it doesn't seem to pose a problem with testing. though I could be overlooking something

Reply:
moving this back !